README.md

```diff
@@ -1,39 +1,56 @@
-# These are the instructions to get an M5Stack poesp32 running on ESPHOME and setting the UART2 pins TX and RX to switch a relay.
-# 
-# Device: https://shop.m5stack.com/products/esp32-ethernet-unit-with-poe
-#
-# Edit the M5Stack_poesp32.yaml file and rename the switches to whatever you want to call them.
-# Change the API to one that you generate (you can use https://generate.plus/en/base64 and set the length to 32).
-#
-# Ensure that we have the latest version of ESPHOME
-# update python etc (this is for Mac OSX)
+These are the instructions to get an M5Stack poesp32 running on ESPHOME and setting the UART2 pins TX and RX to switch a relay.
+
+Device: <https://shop.m5stack.com/products/esp32-ethernet-unit-with-poe>
+
+Edit the `M5Stack_poesp32.yaml` file and rename the switches to whatever you want to call them.
+Change the API to one that you generate (you can use <https://generate.plus/en/base64> and set the length to 32).
+
+Ensure that we have the latest version of ESPHome, Python, and other packages[^1]:
+
+```shell-session
 brew update
 brew upgrade
+```
 
-# update pip
+Update `pip`:
+
+```shell-session
 python3 -m pip install --upgrade pip
+```
+
+Update ESPHome:
 
-# update esphome
+```shell-session
 pip3 install -U esphome
+```
```

Comment:
It looks like `esphome` is available via Homebrew, so this section could probably be simplified by removing the Python and `pip` update steps and replacing them with the following:

```shell-session
brew upgrade esphome
```

Alternatively, you could use an install step instead:

```shell-session
brew install esphome
```

The `brew install` should update the package if it is already installed.